WooCommerce/Classes/ViewRelated/Dashboard/StoreStats/StorePerformanceViewModel.swift

```diff
@@ -475,7 +475,7 @@ private extension StorePerformanceViewModel {
         static let thirtyDaysInSeconds: TimeInterval = 86400*30
 
         /// The wait time before the `StoreStatsUsageTracksEventEmitter.interacted()` is called.
-        static let chartValueSelectedEventsDebounce: TimeInterval = 1.0
+        static let chartValueSelectedEventsDebounce: TimeInterval = 0.5
```

Comment:
I believe the new debounce wait time has disrupted the chart's selection and dragging functionalities. The selection is not consistently responsive, and currently, during the dragging of the chart, the following issue is occurring:



<img src="https://github.com/woocommerce/woocommerce-ios/assets/495617/618acfc6-ee8d-4f51-95e3-c5e3c6c294d1" width=300 />

This is something I didn't experienced before.

Reply:
@pmusolino In a previous PR review you [pointed](https://github.com/woocommerce/woocommerce-ios/pull/12483#discussion_r1567980468) out that the constant updates causes the UI to flicker, so I had to debounce the update to minimize that. Now I'm confused about your suggestion in this comment. Should we disable dragging on the chart to simplify the problem? I don't see the point of supporting that.

Reply:
After reviewing your [most recent update](https://github.com/woocommerce/woocommerce-ios/commit/e8661b29c9de0e63fe82c5d4b0a7f5c105f58954), I noticed that the dragging functionality was working well for me. However, it appears that reducing the debounce time has introduced this new issue. Previously, the problem manifested as the chart continuously refreshing and displaying data. Now, the cursor moves around the chart but fails to select the interval.

Reply:
Btw, as you suggested, I don't see much value in it 👍

Reply:
On a second thought I found a different way to handle the drag gesture on the chart. In b060151553b6b73c0bd39288dc50bf7ad585a441 I updated the selection only after the drag ends, but I still update the chart when the drag changes. LMK what you think!

Reply:
It works nicely! Thanks Huong 🚢